src/test/scala/io/iohk/ethereum/ledger/LedgerSpec.scala

```diff
@@ -62,11 +62,13 @@ class LedgerSpec extends FlatSpec with PropertyChecks with Matchers {
 
       val header = defaultBlockHeader.copy(beneficiary = minerAddress.bytes)
 
-      val mockVM = new MockVM(createResult(_, gasUsed, defaultGasLimit, gasRefund, error))
+      val mockVM = new MockVM(createResult(_, execGasUsed, defaultGasLimit, gasRefundFromVM, error))
       val ledger = new LedgerImpl(mockVM, blockchainConfig)
 
-      val postTxWorld = ledger.executeTransaction(stx, header, initialWorld).worldState
+      val execResult = ledger.executeTransaction(stx, header, initialWorld)
+      val postTxWorld = execResult.worldState
 
+      execResult.gasUsed shouldEqual gasUsed
```

Comment:
well... it seemed to me that too but as it's related to gas thought I might not be so different (a separate test would have almost the same code).

I will split it.